src/Raven.Server/Config/Categories/StorageConfiguration.cs

```diff
@@ -85,6 +85,15 @@ public class StorageConfiguration : ConfigurationCategory
         [ConfigurationEntry("Storage.SyncJournalsCountThreshold", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public int SyncJournalsCountThreshold { get; set; }
 
+        /// <summary>
+        /// Specifies the time interval between each IoMetrics Cleaner run
+        /// </summary>
+        [Description("Time (in hours) between IO Metrics cleanup")]
+        [DefaultValue(24)]
+        [TimeUnit(TimeUnit.Hours)]
+        [ConfigurationEntry("Storage.IoMetricsCleanupIntervalInHrs", ConfigurationEntryScope.ServerWideOrPerDatabase)]
+        public TimeSetting IoMetricsCleanupIntervalInHrs { get; set; }
```

Comment:
property does not have to have 'InHrs' in contract to configuration key. It is a TimeSetting with TimeSpan value

Reply:
Done